rclpy/rclpy/context.py

```diff
@@ -111,6 +111,7 @@ def try_shutdown(self):
             if self.__context.ok():
                 self.__context.shutdown()
                 self._call_on_shutdown_callbacks()
```

Comment:
same here, is this also okay to issue user callback with `_lock`? i think any system lock should not be acquired during user callback.

Reply:
this was already the case